app/src/main/java/org/mozilla/focus/Components.kt

```diff
@@ -130,8 +132,13 @@ class Components(
     }
 
     val store by lazy {
+        val browserStoreMiddlewares = if (Features.SHOW_ERASE_CFR) {
```

Comment:
nit: the actual browser store middlewares are a bit below and there is a list of them.
Can this property be named just "cfrMiddleware" and then be added after engineMiddleware - just to not do the addition both at the beginning and at the end of where the middlewares are set?